common/src/java/com/zimbra/common/gql/GqlConstants.java

```diff
@@ -52,4 +52,116 @@
 
     // end session constants
     public static final String CLEAR_COOKIES= "clearCookies";
+
+    // search constants
+    public static final String SEARCH_PARAMS = "params";
+    public static final String SEARCH_REQUEST = "SearchRequest";
+    public static final String HEADERS = "headers";
+    public static final String INCLUDE_MEMBER_OF = "includeMemberOf";
+    public static final String MSG_CONTENT = "msgContent";
+    public static final String CURSOR = "cursor";
+    public static final String LOCALE = "locale";
+    public static final String CALENDAR_TIME_ZONE = "calTz";
+    public static final String OFFSET = "offset";
+    public static final String LIMIT = "limit";
+    public static final String FIELD = "field";
+    public static final String FULL_CONVERSATION = "fullConversation";
+    public static final String RESULT_MODE = "resultMode";
+    public static final String PREFETCH = "prefetch";
+    public static final String WANT_RECEPIENTS = "wantRecipients";
+    public static final String NEUTER_IMAGES = "neuterImages";
+    public static final String NEED_CAN_EXPAND = "needCanExpand";
```

Comment:
This naming scheme doens't observe changes done in other places. For instance the property `canExpand` -> `isExpandable`. Again, can update later.